crystalball.gemspec

```diff
@@ -43,7 +43,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pry-byebug'
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
-  spec.add_development_dependency 'rubocop'
+  spec.add_development_dependency 'rubocop', "~> 0.56"
```

Comment:
why should we add it? Lets be state-of-the-art always :)

Reply:
I can change it to `>`, because since we have `MethodMissingSuper` config, which is only available after https://github.com/rubocop-hq/rubocop/pull/5811, there will be warnings and inconsistent behavior on CI/locally